examples/esp_idf/lightdb_stream/README.md

```diff
@@ -0,0 +1,209 @@
+# Golioth LightDB Stream sample
+
+## Overview
+
+This LightDB Stream application demonstrates how to connect with Golioth
+and periodically send data to LightDB Stream. In this sample temperature
+value is generated from 20 up to 30 and temperature measurements are sent
+to `/sensor/temp` LightDB Stream path.
+
+## Requirements
+
+* Golioth credentials
+* Network connectivity
+
+## Building and Running
+
+### Command-line
+
+First, setup the environment. This step assumes you've installed esp-idf
+to `~/esp/esp-idf`. If you haven't, follow the initial steps in
+examples/esp_idf/README.md
+
+```sh
+source ~/esp/esp-idf/export.sh
+```
+You may have to set target based on the ESP32 chip you are using.
+For example, if you are using ESP32-C3, enter this:
+
+```
+idf.py set-target esp32c3
+```
+
+Next, `cd` to the lightdb_stream example where you can build/flash/monitor:
+
+Make sure that CONFIG_GOLIOTH_LIGHTDB_STREAM is set by going into the interactive menuconfig
```

Comment:
This is set in `sdkconfig.defaults` so we don't need instructions on setting it through menuconfig here.